source/use-cases/home-resolver.rst

```diff
@@ -1,6 +1,200 @@
+While just changing this file will work as long as the machine doesn't reboot, the more permanent and better solution is to replace the file with our own. The reason for this is that the :file:`resolv.conf` file is a `symbolic link`. We will remove the link and create a new file ourselves.
+
+.. code-block:: bash
+
+	rm /etc/resolv.conf
+
+With your favourite text editor (if you don't have a favourite you could use :command:`nano`), we can then create a new file with the same name and fill it with the IP address that our Unbound instance is running at, and we include the :option:`edns0` option as this enables header extensions used in DNSSEC and is an overall standard used in DNS nowadays. |br|
+So with :file:`nano /etc/resolv.conf` we create the new file and enter:
+
+.. code-block:: bash
+
+	nameserver 127.0.0.1
+	options edns0
+
+We then need to stop and disable the currently running pre-installed resolver. Note that you cannot go to new websites until the next step after this, as you have no DNS resolver assigned for the system.
+
+.. code-block:: bash
+
+	sudo systemctl disable systemd-resolved.service
+	sudo systemctl stop systemd-resolved
+
+Now the operating system should use our Unbound instance as default. A quick test a :command:`dig` without specifying the address of the Unbound server should give the same result as specifying it did above (with ``@127.0.0.1``).
+
+.. code-block:: bash
+
+	dig example.com
+
+Note that the "SERVER" section in the output from :command:`dig` should also contain the local IP address of our server.
+
+.. code-block:: bash
+
+	;; SERVER: 127.0.0.1#53(127.0.0.1)
+
+Setting up for the rest of the network
+--------------------------------------
+
+While we currently have a working instance of Unbound, we need it to be reachable from within our entire network. With that comes the headache of dealing with (local) IP addresses. It’s likely that your home router distributed local IP addresses to your devices. If this is the case (i.e. you didn’t change it by hand), they should be :rfc:`1918` ranges:
+
+.. code-block:: bash
+
+	10.0.0.0 - 10.255.255.255 (10/8)
+	172.16.0.0 - 172.31.255.255 (172.16/12)
+	192.168.0.0 - 192.168.255.255 (192.168/16)
+
+To find the IP address of the machine that is running Unbound, we use:
+
+.. code-block:: bash
+
+	hostname --all-ip-addresses
+
+If you just have one IP address as output from the :command:`hostname` command that will be the correct one. If you have multiple IP addresses the easiest way which IP address to use, is to find out which connection goes to your home router. Keep in mind that finding the wrong IP address here this can be a source of connectivity errors further on. For purpose of this tutorial we imagine that our home router has ``10.0.0.1`` as IP address, and our resolver machine (the machine that is running our Unbound instance) has ``10.0.0.2``, which we will get into in the next section.
```

Comment:
s/the easiest way which IP address/the easiest way to determine which IP address/
s/finding the wrong IP address here this can/finding the wrong IP address here can/
s/For purpose/For the purpose/
s/has 10.0.0.1 as IP address/has the IP address 10.0.0.1/
s/has 10.0.0.2/has IP address 10.0.0.2/